### DISCUSSION TITLE
Mad Max furry road question

### DISCUSSION THREAD

**Discussion by Wilkugmo** [madMAx43v3r/chia-plotter#615] [Q&A]
I recently started using Mad Max plotter for windows but it keeps randomly crashing (at P1, P2 and P3) it just stops and only returns " the press any key to continue . . . ".

I added

```
:start
goto start
```
so it restarts after stopping but it's just a band aid not a solution.

Does anyone know what is the problem here? What can I do?

Any help is appreciated.

My setup:

- Ryzen 7 3800X
- 4x8GB HyperX Fury 3466MHz Cl 16
- Msi X570-A Pro
- Samsung 980 Pro 2tb

My config:

```
:start
"D:\xyz\Programy\Mad_Max_Plotter\plot_with_log.cmd.lnk"
@ECHO OFF
SET hr=%time:~0,2%
IF "%hr:~0,1%" equ " " set hr=0%hr:~1,1%
SET DATETIME=Log_%date:~-4,4%%date:~-10,2%%date:~-7,2%_%hr%%time:~3,2%%time:~6,2%
SET LOG_FILE=logs/%DATETIME%.log
IF NOT EXIST logs MKDIR logs
powershell "D:\xyz\Programy\Mad_Max_Plotter\chia_plot.exe -n -1 -r 8 -u 256 -t G:\TempPlots\ -d G:\MovingPlots\ -p *PUBLIC KEY* -f *FARMER KEY* | tee '%LOG_FILE%'"
goto start
```

**Comment by wabragg** [comment]
I was having unhandled exceptions mostly in phase 1.  It seems to beat the hell out of memory in phase 1, so I added a bit of voltage to my dram banks and that seems to have helped stabilize it.  I'm also on the 0.0.6 version now, updated yesterday.

**Reply by Wilkugmo** [reply]
So you're saing to just increase ram voltage a bit in Bios?

**Reply by wabragg** [reply]
Yes.  My XMP setting was 1.35v, I just bumped it to 1.45v.

**Reply by Wilkugmo** [reply]
Ok complete stranger from the internet, I will use your advise and randomly increase power in my PC component :D

For real tho, thanks. I already lowered cores used by Madmax from -r 8 to -r 4 and it seems to work for now, so it might just be my memory.

**Reply by wabragg** [reply]
I've never had a stability problem in the past using the basic XMP settings but I've never really stressed things too very hard either.  I've got a 3200CL14 kit that I was bumping to 3600CL15 with voltage bump but that wasn't stable under this plotter.  I've rolled back to the base settings but just kept to 1/10th of a volt bump and it seem to run well with reasonable temps.  I had heard this suggested in another forum and it seems to have worked for me which is why I passed it along.

**Reply by wabragg** [reply]
To clarify, you might bump 1/10th of a volt and not use the absolute 1.45v.  If you're running at 1.2v currently, maybe bump to 1.3v.

**Reply by Wilkugmo** [reply]
Thanks, also did you see faster plotting time with OC (unstable) memory?
Now when i think about it it's probably cause I have 4 sticks of RAM (which very often can be unstable at higher speeds) Also, with parallel plotting I needed 32GB ram for my system but now Madmax uses only up to 11GB. I can still return 2x8GB ram so I might just do that and OC my older 2x8GB ram (Or just leave it at XMP if it does not matter)


EDIT:
I found out that XMP already raised my voltage to 1.365V. I changed it to 1.4V, my PC has great air flow so I think it will not be a problem. I also started plotting with Madmax again, hopefully raising voltage solved the issue but I will know soon anyway.

**Reply by wabragg** [reply]
I was unable to get better throughput than the 3200CL14 XMP settings.  I tried several variations including 3600CL15 @1.5v and it tested unstable when stressed using the MemTest in the DRAM-Calculator for Ryzen (even though I'm running Intel).  So I'm just running my XMP setting with that small voltage bump and that has given me stability.  I'm also using 4 sticks with the 4 channel controller in my HEDT processor.

**Reply by Wilkugmo** [reply]
I solved it, I found out that usually you should change command rate from 1T to 2T if you have 4 sticks. Tried it and now madmax works fine.
Also changing T1 to T2 is like going from CL16 to CL17.

**Reply by wabragg** [reply]
Nice find.  I've got 4 sticks but mine was already 2T.  Still, I needed the voltage bump to be stable even on my base XMP setting.  My machine has probably been that way since I built it but I've never hit memory hard enough for it to show the issue until this. I must have a poor kit of G.Skill.  AIDA64 testing didn't show the problem but the MEMbench in the DRAM Calculator caught it.
